showcases/shared/notification.json

```
@@ -252,7 +303,18 @@
 					"img": true,
 					"variant": "standalone",
 					"behaviour": "permanent"
-				}
+				},
+				"children": [
+					{
+						"name": "img",
+						"slot": "image",
+						"content": "",
+						"props": {
+							"src": "/assets/images/placeholder.jpg",
+							"alt": "placeholder"
```

```suggestion
							"alt": ""
```

let's leave this empty, as it's a decorative graphic.